app/javascript/mastodon/selectors/accounts.ts

```diff
@@ -0,0 +1,88 @@
+import type { Map as ImmutableMap } from 'immutable';
```

Comment:
The motivation behind this was not because it's nicer (it is), but because it appears importing anything from a js file causes my vscode linter to assume the type is `any`, even when it's not :(